osd/customer_pod_preventing_drain.json

```diff
@@ -0,0 +1,7 @@
+{
+ "severity": "Warning",
+ "service_name": "SREManualAction",
+ "summary": "Action required: Pod(s) preventing Node Drain",
+ "description": "Your cluster is attempting to drain a node but there are pod(s) preventing the drain. The SRE team has identified the pod(s) as '${POD}' running in namespace(s) '${NAMESPACE}'. Please re-schedule the impacted pod(s) so that the node can drain. If you require assistance, please file a support request.",
```

Comment:
the default template already contains the statement to file a support ticket

Reply:
fixed thanks!